### PR TITLE
Steakwallet => Omni

### PR DESCRIPTION
**Pull request by AlexBHarley**
We've rebranded

### CODE REVIEW

File: packages/rainbowkit/src/wallets/walletConnectors/omni/omni.ts [hunk 1]
```diff
@@ -60,3 +59,6 @@ export const steak = ({ chains }: SteakOptions): Wallet => ({
     };
   },
 });
+
+// backwards compatibility
+export const steak = omni;
```
Comment (jxom) [inline packages/rainbowkit/src/wallets/walletConnectors/omni/omni.ts:64]: Perhaps we can add a `/** @deprecated */` tag here?

**Comment by jxom** [comment]
Can we add a changeset for this? 🙏 

`pnpm changeset`

[Example changeset](https://github.com/rainbow-me/rainbowkit/blob/0b9849733e34d8e8b6498f9dbc4b001d0f5669a6/.changeset/hip-clocks-serve.md)

File: packages/rainbowkit/src/wallets/walletConnectors/omni/omni.ts [hunk 2]
```diff
@@ -4,19 +4,19 @@ import { isAndroid } from '../../../utils/isMobile';
...
-  name: 'Steakwallet',
-  iconUrl: async () => (await import('./steak.svg')).default,
+export const omni = ({ chains }: OmniOptions): Wallet => ({
+  id: 'omni',
+  name: 'Omni',
+  iconUrl: async () => (await import('./omni.svg')).default,
   iconBackground: '#000',
   downloadUrls: {
     android: 'https://play.google.com/store/apps/details?id=fi.steakwallet.app',
     ios: 'https://apps.apple.com/np/app/steakwallet/id1569375204',
```
Comment (peduarte) [inline packages/rainbowkit/src/wallets/walletConnectors/omni/omni.ts:18]: should the `android` and `ios` download URLs also be updated? or is the rebranded app not published yet?

Comment (AlexBHarley) [reply packages/rainbowkit/src/wallets/walletConnectors/omni/omni.ts]: Another good catch, iOS yes but Android no. We didn't create a new app, so we'll always maintain the steakwallet bundle ID

**Review by jxom — APPROVED** [review]

**Review by peduarte — APPROVED** [review]
lgtm!

**Pull request merged**